react/components/admin/institutional/Form/withContentContext.tsx

```diff
@@ -103,9 +106,44 @@ function withContentContext<T>(
                   return <Loader />
                 }
 
-                const blockId = dataTemplates!.availableTemplates.filter(
+                const elegibleTemplates = dataTemplates!.availableTemplates.filter(
                   (template: Template) => template.id !== interfaceId
-                )[0].id
+                )
+
+                if (!elegibleTemplates.length) {
+                  return (
+                    <div>
+                      <EmptyState
+                        title={
+                          <FormattedMessage
+                            id="admin/pages.admin.content.not-supported.title"
+                            defaultMessage="This feature is not available :/"
+                          />
+                        }
+                      >
+                        <p>
+                          <FormattedMessage id="admin/pages.admin.content.not-supported.description" />
+                        </p>
+                        <div className="pt5">
+                          <Button variation="secondary" size="small">
+                            <a
+                              className="link c-action-primary"
+                              href="https://github.com/vtex-apps/admin-pages/blob/next/react/components/admin/institutional/README.md"
```

Comment:
Ok, I since `next` is the new `master`, point this to `master` instead of `next`... Also don't forget to update the path if you move the file to `docs/`